src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/createnewwiki/CreateNewWikiPageObjectStep1.java

```diff
@@ -57,15 +56,20 @@ public String getWikiName() {
 
 
   public void selectLanguage(String lang) {
-    scrollAndClick(languageSelectorTrigger);
-    wait.forElementVisible(languageSelector);
-    Select language = new Select(languageSelector);
-    List<WebElement> langList = language.getOptions();
+    wait.forElementVisible(wikiLanguageDropdown);
+    wikiLanguageDropdown.click();
+
+    List<WebElement> langList = wikiLanguageList.findElements(By.cssSelector("li:not(.spacer)"));
+
     for (int i = 0; i < langList.size(); i++) {
-      String langDropElement = langList.get(i).getText();
-      if (langDropElement.contains(lang + ":")) {
-        language.selectByIndex(i);
+      WebElement selectedLanguage = langList.get(i);
+      String selectedLanguageText = selectedLanguage.getText();
+      if (selectedLanguageText.contains(lang + ":")) {
```

Comment:
can we move this string concatenation outside of loop to not concatenate it with every iteration?